Source/WebKit/WPEPlatform/wpe/wayland/WPEWaylandCursor.cpp

```diff
@@ -31,10 +31,19 @@
 
 namespace WPE {
 
-WaylandCursor::WaylandCursor(WPEDisplayWayland* display)
+std::unique_ptr<WaylandCursor> WaylandCursor::create(WPEDisplayWayland* display)
+{
+    auto theme = WaylandCursorTheme::create(wpe_display_wayland_get_wl_shm(display));
+    if (theme)
+        return makeUnique<WaylandCursor>(display, WTFMove(theme));
+    g_warning("Could not load cursor theme, disabling custom cursors support");
+    return nullptr;
+}
```

Comment:
I'm, not sure we want this, we are already handling the case of the theme being nullptr in WaylandCursor, with this patch we don't allow to set custom cursors when the theme fails to load, for example. I think we should avoid setting the cursor when it was set by a name and the theme failed to load. It would be interesting to know why the theme failed to load, and maybe we could even add a few embedded cursors and have a simple fallback theme to ensure we always have cursors